### ISSUE TITLE
Skip emails already exported like sync

### ISSUE THREAD

**Issue by abides** [gaubert/gmvault#294]
Hello,

I'm trying to create backup flow and my purpose is keep backed up emails reachable anytime.

So i have a mail server with dovecot and i have webmail (roundcube).

First i backup all gmail account to local folder without clean up option then i export everything to dovecot and i can easily see all my backed up emails using local webmail. Then i can delete old emails from my gmail. (That is why i need archive accesible anytime)

Bu i need to create schedule (cron) to backup and export regularly to keep backup alive. Its easy with quick sync to get last 7 days emails to folder but exporting is always start at the begining and take long time to complete. (im not sure if its overwriting or recreating the email, didnt tested that)

Is there a skip option when exporting? or export just last 7 days with overwrite...etc. If not, is there any suggestion to achieve this regular live backup?

**Comment by abides** [comment]
I would like to contribute this but im not familiar with python. Some ideas on my mind;

- While exporting, mark email in the DB as "exported" somehow (add field in the meta or else)
- Option to get nonmarked emails in the DB
- Option to reset "exported" mark in the DB (in case of something goes wrong and need to export everything again.

Best regards.

**Comment by gaubert** [comment]
@abides didn't know somebody was using the export function. I would need some help to support it and extend it. Would you like to help me ?

**Comment by abides** [comment]
@gaubert  Of course, I want to help, what do you need? (Except programming in python :)) 

I thought people mostly use this for open space in their mailbox when it's getting full since I newer heard Gmail data loss before. (I'm using Gmail more than 10 years). Or maybe switch another mail solution? Which still require export.
